platforms/ubuntutouch/clickable.yml

```diff
@@ -0,0 +1,24 @@
+clickable_minimum_required: "7.6"
+builder: cmake
+kill: gcompris-qt
+build_args: "-DUBUNTU_TOUCH=ON"
+dependencies_target:
+  - libqt5charts5-dev
```

Comment:
I think you also need to install `qml-module-charts` else fractions activities won't work

Reply:
this is also something missing in the build cli of ubuntu touch. i need to add the dev package whereas the module is already installed on devices